packages/web/pages/_app.tsx

```diff
@@ -216,6 +216,12 @@ const MainLayoutWrapper: FunctionComponent<{ children: ReactNode }> = observer(
           icon: <Icon id="pool" className="h-5 w-5" />,
           selectionTest: /\/pools/,
         },
+        {
+          label: "Smart Account",
```

Comment:
Reminder to add localization

Reply:
Thank you for the feedback! This route will only be used for testing. We'll most likely remove it on release